vmdb/Gemfile

```diff
@@ -42,7 +42,13 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 gem "default_value_for",              "~>2.0.3"
-gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+
+if RUBY_PLATFORM == 'java'
+  gem "puma"
+else
+  gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+end
```

Comment:
I am good with switching servers, and I agree now is the time.  We had discussed unicorn, but puma is good too.  What about passenger/raptor?  The biggest question mark is how it plays with the ui/web service workers as @jrafanie mentioned.  Maybe we don't bother with that kind of configuration anymore, or just do it completely differently.